commands/import.go

```diff
@@ -91,6 +102,9 @@ func init() {
 	ImportJobListCmd.Flags().Int("per-page", 200, "Number of import jobs to be fetched")
 	ImportJobListCmd.Flags().Bool("all", false, "Fetch all import jobs. --page flag will be ignore if provided")
 
+	ImportValidateCmd.Flags().StringArray("team", nil, "The names of the team[s] (flag can be repeated)")
```

Comment:
Shall we document, this can be omitted? I think this still works if we don't specify any team right?